### PR TITLE
Add support for import command

### LINKED ISSUE #42: Add support for terraform import
Below the monkey patch I use:

```
#   Import existing infrastructure into your Terraform state.
#
#   This will find and import the specified resource into your Terraform
#   state, allowing existing infrastructure to come under Terraform
#   management without having to be initially created by Terraform.
#
#   The ADDR specified is the address to import the resource to. Please
#   see the documentation online for resource addresses. The ID is a
#   resource-specific ID to identify that resource being imported. Please
#   reference the documentation for the resource type you're importing to
#   determine the ID syntax to use. It typically matches directly to the ID
#   that the provider uses.

module RubyTerraform
  module Commands
    class Import < Base
      def configure_command(builder, opts)
        path = opts[:path] || opts[:directory]
        json_format = opts[:json]
        no_color = opts[:no_color]
        module_depth = opts[:module_depth]

        builder
          .with_subcommand('import') do |sub|
          sub = sub.with_option('-module-depth', module_depth) if module_depth
          sub = sub.with_option('-config', path) 
          sub = sub.with_flag('-no-color') if no_color
          sub = sub.with_flag('-json') if json_format
          sub
        end
          .with_argument(opts[:addr])
          .with_argument(opts[:id])
      end
    end
  end
  module ClassMethods
    def import(opts = {})     
     Commands::Import.new.execute(opts)
    end
  end
end

```

### PR DESCRIPTION
**Pull request by nxn128**
Adding support for the `terraform import` command.

Closes #42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,6 +242,11 @@ The destroy command supports the following options passed as keyword arguments:
   `var_files` are provided, all var files will be passed to terraform.
 * `var_files`: an array of paths to terraform var files; if both `var_file` and
   `var_files` are provided, all var files will be passed to terraform.
+  * `module_depth`: the depth of modules to show in the output; defaults to
+  showing all modules.
+* `json`: whether or not the output from the command should be in json format;
+  defaults to `false`.
+
```
Comment (tobyclemson) [inline README.md:249]: This looks like a copy paste error as the destroy command doesn't take these options.

Comment (nxn128) [reply README.md]: whoops... sorry about that.

**Comment by tobyclemson** [comment]
Hi Nick,

ruby-terraform [0.58.0](https://rubygems.org/gems/ruby-terraform/versions/0.58.0) includes this change.

Thanks,
Toby

**Comment by chussenot** [comment]
Thx @nxn128

**Pull request merged**